src/deps.ts

```diff
@@ -90,9 +90,8 @@ const builtins = new Set([
   'zlib',
 ])
 
-const nameRe =
-  /^(?<name>(@[a-z0-9-~][a-z0-9-._~]*\/)?[a-z0-9-~][a-z0-9-._~]*)\/?.*$/i
-const versionRe = /^@(?<version>[~^]?(v?[\dx*]+([-.][\d*a-z-]+)*))/i
```

Comment:
The fix here makes the regex less precise, but I don't think that's a problem? I think it should be good enough to catch false positives?

[The issue](https://redosdetector.com/?pattern=%5E%40%28%5B%7E%5E%5D%3F%28v%3F%5B%5Cdx*%5D%2B%28%5B-.%5D%5B%5Cd*a-z-%5D%2B%29*%29%29&caseInsensitive=true&unicode=false) was in the second part.